helmfile.d/snippets/defaults.yaml

```diff
@@ -187,6 +187,16 @@ environments:
         dns:
           domainFilters: []
           zoneIdFilters: []
+        team:
```

Comment:
These are otomi defaults not a tests/fixtures ;)

Reply:
removed